#! /bin/sh /usr/share/dpatch/dpatch-run
## perl-uninitialized-var.dpatch by Ulrich Habel <rhaen@netbsd.org>
##
## DP: Fixed an uninitialized variable warning.

@DPATCH@

diff a/src/perl.c b/src/perl.c
--- a/src/perl.c
+++ b/src/perl.c
@@ -1790,7 +1790,7 @@ static int perl_config (oconfig_item_t *ci)
 
 	for (i = 0; i < ci->children_num; ++i) {
 		oconfig_item_t *c = ci->children + i;
-		int current_status;
+		int current_status = 0;
 
 		if (NULL != perl_threads)
 			aTHX = PERL_GET_CONTEXT;
